applications/conf/routes

```diff
@@ -18,6 +18,10 @@ GET        /crosswords/accessible/$crosswordType<cryptic|quick|quiptic|prize|eve
 GET            /crosswords/search                                                                                                controllers.CrosswordSearchController.search()
 GET            /crosswords/lookup                                                                                                controllers.CrosswordSearchController.lookup()
 
+# Email paths
+GET        /email                                                               controllers.EmailController.renderPage()
+POST       /email                                                               controllers.EmailController.submit()
```

Comment:
Good point. I asked Grant and we should be using redirects rather than 200 responses from POSTs. In this case, our responses don't display anything unique to the post data, so the success/error pages we redirect to _can_ be cached.